04_numeronym/nicholeL/main_test.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"bytes"
+
+	"github.com/stretchr/testify/assert"
+
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := strconv.Quote("[a11y K8s abc]")
+	actual := strconv.Quote(buf.String())
+	if actual != expected {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+func TestNumberShorten(t *testing.T) {
+	testCases := map[string]struct {
+		input    string
+		expected string
+	}{
+		"Happy case":                {"accessibility", "a11y"},
+		"length of string is three": {"asd", "asd"},
```

Comment:
Add Unicode characters like emojis - you'll have to update your code to work with runes.

Reply:
thanks, will add unicode characters like emojis